RealFlight/Released_Models/QuadPlanes/Titan_Cobra_VTOL/README.md

```diff
@@ -0,0 +1,37 @@
+# Titan Dynamics Cobra VTOL
+
+Aircraft Model: Titan Cobra VTOL
+
+![JPG](https://github.com/robertlong13/SITL_Models/raw/titan_cobra_vtol/RealFlight/Released_Models/QuadPlanes/Titan_Cobra_VTOL/TitanCobraVTOL.jpg)
```

Comment:
This line will need to be changed to point to its final destination before merging